swim/ping_request_sender.go

```diff
@@ -112,6 +115,26 @@ func (p *pingRequestSender) MakeCall(ctx json.Context, res *pingResponse) <-chan
 	return errC
 }
 
+func indirectHealthCheck(n *Node, target string, size int, timeout time.Duration) (bool, []error) {
```

Comment:
call it indirectPing. Let's not introduce a new term when there doesn't need to be.